app/javascript/components/pages/Show.js

```diff
@@ -1,13 +1,49 @@
 import React, { Component } from 'react'
+import { NavLink } from 'react-router-dom'
+import { Button } from 'reactstrap'
 
-class Show extends Component {
-  render() {
-    return(
-      <>
-        <h1>Hello this is the show page!</h1>
-      </>
-    )
+  class Show extends Component {
+    constructor(props){
+      super(props)
+        this.state= {
+          newEvent: {
+            image: "",
+            category: "",
+            about: "",
+            event_name: ""
+          },
+          submitted: false
+        }
+      }
+
+    render() {
+
+      console.log("propsforshowjs",this.props)
```

Comment:
Putting a console.log in a method while you're deving is not optimal but it's okay. But putting a console log in the render is no bueno. You shouldn't commit console logs.